tests/ui-toml/ifs_same_cond/clippy.toml

```
@@ -0,0 +1 @@
+ignore-interior-mutability = ["std::cell::Cell"]
```

Could you add a new line at the end of this file?
```suggestion
ignore-interior-mutability = ["std::cell::Cell"]

```

---

Done~